### ISSUE TITLE
Add nbclassic support

### ISSUE THREAD

**Issue by Koncopd** [laminlabs/nbproject#273]
subj

**Comment by Koncopd** [comment]
Yep, i agree and will check them. But this i need mostly for myself, it is also not hard to add at all.

btw i checked and we have interactivity for jupyter lab and new notebook 7 via ipylab. So the only missing jupyter frontend is nbclassic now.

**Comment by falexwolf** [comment]
These are good news! :D
